added windows setup (#4)
lepsta/docs: docs/pages/setup.md

=== "Windows"

>     !!! warning "We are working on the Windows version."

Reviewer: ```suggestion
    !!! warning "We are still working on the Windows version."
```

        We are still working on the Windows version and will make an announcement as soon as it is available.

>         However to get it working on windows you must activate the Windows Subsystem for Linux.

Reviewer: ```suggestion
        However, as a workaround, you can get it working on windows by activating the Windows Subsystem for Linux.
```

    You must be on Windows 10 to activate Windows Subsystem for Linux for Uju to work.

>     

Reviewer: Consider separating this into steps. For example, this next section can have a heading like:

### Enable Windows Subsystem

The next section can then be:
### Setup Uju in your Linux distro

    - Click the OK button.
    - Click the Restart now button.
    Once you complete the steps, the environment will be configured to download and run the distros of Linux on Windows 10.

    Head on to <a href="https://docs.microsoft.com/en-us/windows/wsl/install-win10https://docs.microsoft.com/en-us/windows/wsl/install-win10" target="_blank"> Microsoft Docs </a> to complete the remaining steps or to <a href="https://www.windowscentral.com/install-windows-subsystem-linux-windows-10" target="_blank"> Windows central </a> for more detailed instructions. 

    !!! Note "Note"
        If you are running Windows 10 version 1903 or 1909, open "Settings" from your Windows menu, navigate to "Update & Security" and select "Check for Updates". Your Build number must be 18362.1049+ or 18363.1049+, with the minor build # over .1049. Read more: WSL 2 Support is coming to Windows 10 Versions 1903 and 1909.
        
        See the troubleshooting instructions on  <a href="https://docs.microsoft.com/en-us/windows/wsl/install-win10https://docs.microsoft.com/en-us/windows/wsl/install-win10" target="_blank"> Microsoft Docs. </a>
    
    

>     Once you are done Installing Linux distros, Launch the linux distros that you installed eg. Ubuntu or Mint.

Reviewer: ```suggestion
    Once you have installed the Linux distro eg. Ubuntu or Mint, launch it.
```
"distro" must be singular

    
    ![Launch the Linux distros](/assets/images/launch.png)

>     - Create a username for the Linux distro and press Enter.
>     - Specify a password for the distro and press Enter.

Reviewer: ```suggestion
    - Create a username for Linux and press Enter.
    - Specify a password and press Enter.
```
Preventing over-se of the word "distro"

    ![Create a username and password](/assets/images/linux.PNG)

    You can install Uju by running one of these commands on your terminal.
        ```
        curl https://lepsta.tech/get | sh
        ```
        or
        ```
        wget -qO- https://lepsta.tech/get | sh
        ```

>     ![Get Uju](/assets/images/downloadUju.PNG)

Reviewer: I don't think we need this especially since we didn't add it to the Linux and Mac side. Just for consistency, I think we should either remove it here or add screenshots dor Linux and Mac as well.

Author: Thank you for these suggestions. This part actually affects te Linux, Mac and windows part. Kindly review again and confirm.
